### PR TITLE
Up-to-date and complete Norwegian YAML translations

### PR DESCRIPTION
**Pull request by ghost**
I was a bit behind on my pull requests, so translators have been duplicating work on the Norwegian version. These files were edited using a new three-way YAML non-destructive merge tool I made. The tool will soon support JSX language files too.

### CODE REVIEW

File: config/locales/activerecord.no.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+en:
```
Comment (ykzts) [inline config/locales/activerecord.no.yml]: `no`?

**Review by ykzts — CHANGES_REQUESTED** [review]
Can you resolve conflicts?

**Comment by thor-the-norseman** [comment]
Uhh, Gargron changed all the translation files. I have no idea what happened to all my strings.

**Comment by ykzts** [comment]
@thor-the-norseman `no.jsx` move to `app/javascript/mastodon/locales`. And, Re-run `yarn run manage:translations`.

**Comment by yookoala** [comment]
@thor-the-norseman I've re-did your work here in bc068bb and started another PR.

There are some unused keys (which I used i18n-tasks to remove) and some missing translations (`admin.accounts.disable_two_factor_authentication`, `errors.429`) left to do.

**Comment by yookoala** [comment]
Instead of merging from upstream, I'd suggest you to do this before continue:
```
git checkout master
git fetch upstream master
git reset --hard upstream master
git push -f origin master
```

This will clear all the messy "merge" and simply synchronize your master to the upstream. Will reduce a hell lot of headache to do similar things otherwise.

Alternatively, you may remove your mastodon repository and re-do a "fork" from upstream.

**Pull request closed without merging**